Userland/Libraries/LibWeb/Bindings/AudioConstructor.cpp

```diff
@@ -41,7 +41,7 @@ JS::ThrowCompletionOr<JS::NonnullGCPtr<JS::Object>> AudioConstructor::construct(
     auto& vm = this->vm();
 
     // 1. Let document be the current global object's associated Document.
-    auto& window = verify_cast<HTML::Window>(HTML::current_global_object());
```

Comment:
In the first commit you updated the copied spec text language to add "principal" for the ESO. Seems a miss here, and in Animations/Animation.cpp.

... and more. Perhaps best to leave that to a tracking issue in html/shadow realms

Reply:
Whoops, that is a miss on my end for this one. It is covered in the open merge request, see: https://whatpr.org/html/9893/b8ea975...32ad7a4/media.html#dom-audio

I did a scan over the spec folders for specs which have an open MR, but missed this one since it's in the Bindings folder. Will do a second scan over and adjust